service-templates/apigw-lambda-svc/v1/instance_infrastructure/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "aws_region" {
```

Comment:
Correct me if I am wrong, but I believe the variables defined here should align 1:1 with the variables in the `schema.yaml` file, correct? I did not update them all completely as I wasn't sure